### ISSUE TITLE
Build OrderSubmitted as a EventSummary

### ISSUE THREAD

**Issue by andrzejkrzywda** [RailsEventStore/ecommerce#221]
The goal is to then have consumers use this event.
Then we can get rid of Ordering::OrderLines because they are only used for the need of this event.

Also, it will help to eliminate some "processes".

**Comment by pjurewicz** [comment]
> Then we can get rid of Ordering::OrderLines because they are only used for the need of this event.

It turns out that there is one invariant that takes advantage of order_lines:
`raise CannotRemoveZeroQuantityItem if @basket.quantity(product_id).zero?`

**Comment by andrzejkrzywda** [comment]
I don't think this is an important feature. We can move it to somewhere else (pricing?) or get rid of it.
